### ISSUE TITLE
CloudShell is not opening

### ISSUE THREAD

**Issue by anupamishra333** [microsoft/studentambassadors#189]
### Describe the bug

CloudShell is not opening. It is showing Sign-in but can't move forward

### What happened?

When I try to open CloudShell upper right corner of a code block. It shows a continuous Sign-in Loading page. I checked all possibilities like the internet but the issue was not solved. Therefore I launch Cloud Shell in a separate browser tab by going to https://shell.azure.com/.

### What did you expect to happen?

It should launch CloudShell

### How would you reproduce the bug?

1. Go to [Create a single database - Azure SQL Database](https://learn.microsoft.com/en-us/azure/azure-sql/database/single-database-create-quickstart?view=azuresql&tabs=azure-portal).

2. Go to PowerShell.
![Screenshot (3)](https://github.com/microsoft/studentambassadors/assets/66557767/d4cd5bc9-e261-482c-b058-8736d230858e)

3.Go to Codeblock and click on PowerShell
![Screenshot (4)](https://github.com/microsoft/studentambassadors/assets/66557767/ed9f7fa9-8219-46f7-a7fb-d7b1a038b03b)

### Did this happen via Desktop and/or Smartphone?

Desktop

### Operating System

Windows 11

### What browsers are you seeing the problem on?

Chrome

### What is your current role?

Student Ambassador

### What technical topic is this bug related to?

Application Development, Other

**Comment by paladique** [comment]
@anupamishra333 thank you for this issue, do you experience this on any other module that has a sandbox environment?

**Comment by anupamishra333** [comment]
No, I just tried on another module and works well on another module
